pennylane/measurements/measurements.py

```diff
@@ -90,6 +91,9 @@ def __repr__(self):
 VnEntropy = ObservableReturnTypes.VnEntropy
 """Enum: An enumeration which represents returning Von Neumann entropy before measurements."""
 
+VnEntanglementEntropy = ObservableReturnTypes.VnEntanglementEntropy
```

Comment:
It's probably too late to deprecate `ObservableReturnTypes` for this release but let's remember to include it in the next release? @albi3ro

Reply:
cc @isaacdevlugt @Alex-Preciado

Reply:
@astralcai I believe there's already a story for it in the 0.40 deprecations. See story 71892